src/server/board.ts

```diff
@@ -24,18 +24,26 @@ export class Board {
     const layout = this.boardLayout;
     let can = false;
     // don't let Nick S see this, he hates switch statements
-    switch(direction) {
+    switch (direction) {
       case 'up':
-        if ((y - 1 in layout) && (layout[y - 1][x].p)) can = true;
+        if ((y - 1 in layout) && (layout[y - 1][x].p)) {
```

Comment:
I think we should use brackets for if/else statements.  It makes it easier to extend if you want to put more logic in the block and it reduces minification errors.